selenium/che-selenium-test/src/test/resources/templates/workspace/docker/default.json

```diff
@@ -9,21 +9,7 @@
             "org.eclipse.che.exec"
           ],
           "attributes": {
-            "memoryLimitBytes": "desired_memory_value"
-          },
-          "servers" : {
-            "tomcat8" : {
-              "port" : "8080",
-              "protocol" : "http"
-            },
-            "tomcat8-debug" : {
-              "port" : "8000",
-              "protocol" : "http"
-            },
-            "codeserver" : {
-              "port" : "9876",
-              "protocol" : "http"
-            }
+            "memoryLimitBytes": "2147483648"
```

Comment:
Why do decided to make it impossible to set `desired_memory_value` automatically through the injection?